### PR TITLE
Use Atom editor for examples

### PR DESCRIPTION
**Pull request by robinpokorny**
I think we should not promote Sublime text as it is a paid software (with free evaluation, yes).

Atom by GitHub is open source and free. It provides nearly the same features.

### CODE REVIEW

File: _posts/2015-01-02-first-steps.markdown [hunk 1]
```diff
@@ -40,4 +40,4 @@ Now click `File -> Save as…` and navigate to a folder you know how to find on
...
 
 ###Edit, Save and Refresh workflow
-Leave this browser window open and go back to your text editor. Edit your index.html by adding a second sentence. For example, you might edit your text to look like this: "Hello world! How are you?". Click "File -> Save" again (or use the keyboard shortcut to save some time: CMd + S). Now go back to your browser. No change showing? Sad face? You’ll have to click the "Refresh" button (or use the shortcut: CMD + R) in your browser to see the change on screen. All there? Great! Now let’s make this a real website.
+Leave this browser window open and go back to your text editor. Edit your index.html by adding a second sentence. For example, you might edit your text to look like this: "Hello world! How are you?". Click "File -> Save" again (or use the keyboard shortcut to save some time: CMD + S). Now go back to your browser. No change showing? Sad face? You’ll have to click the "Refresh" button (or use the shortcut: CMD + R) in your browser to see the change on screen. All there? Great! Now let’s make this a real website.
```
Comment (robinpokorny) [inline _posts/2015-01-02-first-steps.markdown]: Here I changed keyboard shortcut on line 3:

```diff
- CMd + S
+ CMD + S
```

**Comment by verpixelt** [comment]
We can add Atom but we won't replace it. Adjust you PR please.

**Comment by robinpokorny** [comment]
@verpixelt, what do you mean by that? Should I include links to both of them?

**Comment by Kriesse** [comment]
+1 on adding more editors, but not removing Sublime.
One downside of Atom is that it requires OSX 10.8 or higher, but attendees sometimes run older versions. Maybe even add http://macromates.com/download as another alternative.

**Comment by verpixelt** [comment]
The course material is easy enough so every coach should be able to handle whatever editor is in front of them. Nevertheless I'd like to keep it down to ST and Atom (so we're comfortable with supporting our attendees).

**Pull request closed without merging**